Report.java

```diff
@@ -202,4 +205,16 @@ private void printSpriteCounts(PrintWriter printW, int i)
             printW.println();
         }
     }   
+
+    /**
+     * Method to get the current datetime for report.
+     *
+     * @return formatted datetime
+     */
+    public String getReportDateTime()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
```

Comment:
I think that would be fine, maybe shorten it a little by making year two numbers though. We'll need to update and let the user know what the report name will be in the read me so they are not surprised. Not sure if telling them what it means will be helpful though.